Nodejs/Product/InteractiveWindow/Repl/Commands/EchoReplCommand.cs

```diff
@@ -15,7 +15,7 @@ public Task<ExecutionResult> Execute(IReplWindow window, string arguments)
             if (string.IsNullOrWhiteSpace(arguments))
             {
                 var curValue = (bool)window.GetOptionValue(ReplOptions.ShowOutput);
-                window.WriteLine("ECHO is " + (curValue ? "ON" : "OFF"));
+                window.WriteLine(string.Format(Resources.echoState, curValue ? "\'ON\'" : "\'OFF\'"));
```

Comment:
Why wouldn't "On" and "Off" be localized?

Reply:
they are actual values you pass to the command, so keeping them as is seems less confusting